client/app/components/add-website/component.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    store: Ember.inject.service(),
+    urlTitle: '',
+    urlAddress: '',
+    urlDescription: '',
+    urlSaveErrors: null,
+    clearInputs () {
+        this.set('urlTitle', '');
+        this.set('urlAddress', '');
+    },
+    actions:{
+        addWebsite (){
+            let self = this;
+            let item = this.get('store').createRecord('item', {
+                title: this.get('urlTitle'),
+                type: 'website',
+                metadata: this.get('urlDescription'),
+                status: 'pending',
+                url: this.get('urlAddress'),
+                source_id: this.get('urlAddress'),
+                collection : this.get('model')
+            });
+            item.save().then(function(){
+                self.transitionToRoute('collection', self.get('model.id'));
```

Comment:
When I add a website, it does not transition back to the collection detail page. Would you mind taking a look?